airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2667,6 +2667,8 @@ components:
         - connectionConfiguration
         - workspaceId
       properties:
+        sourceId:
```

Comment:
This isn't marked required because checks that happen on initial connector setup don't have these IDs at that point in the flow (they haven't been created yet) - so it's totally valid that checks can happen without a set sourceId / destinationId